### PR TITLE
Feature/download file

### PR DESCRIPTION
**Pull request by rob5standingby**
## Why?
We need a function in the sdk to be able to retrieve a temporary file URL for a file stored in the Files API.

## What changed?
- Added a `download` function
- Added tests
- Updated documentation
- Updated `CHANGELOG` with some notes about the new module, and put a placeholder date, `2019-xx-xx` for now until we're ready to publish the new version.

## JIRA
https://ndustrialio.atlassian.net/browse/CFILE-11

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## [v0.0.38](http://github.com/ndustrialio/contxt-sdk-js/tree/v0.0.38) (2019-xx-xx)
+
+**Added**
+
+* Added Files module with the ability to perform read, download on Files stored in the Files API
```
Comment (greg5green) [inline CHANGELOG.md]: Can you list out which endpoints specifically?

File: src/files/index.js [hunk 2]
```diff
@@ -31,6 +37,36 @@ class Files {
     this._sdk = sdk;
   }
 
+  /**
+   * Gets a temporary URL for the file in S3.
```
Comment (greg5green) [inline src/files/index.js]: Can we get rid of this reference to S3 here and anywhere else in this project? The underlying implementation details are not important

Comment (rob5standingby) [reply src/files/index.js]: Good call

Comment (rob5standingby) [reply src/files/index.js]: This is the only reference, removed it.

File: CHANGELOG.md (hunk 1 above)
Comment (rob5standingby) [reply CHANGELOG.md]: On it

**Comment by rob5standingby** [comment]
@greg5green Let me know if this looks good now. Removed the S3 reference, explicitly called out the new functions in CHANGELOG

**Pull request merged**